lib/puppet/application/inspect.rb

```diff
@@ -74,6 +74,8 @@ def help
   end
 
   def setup
+    deprecate
```

Comment:
@Magisus minor comment - can we also add `(Deprecated)` to the help output above?

I.e., so I get something like:
```
be puppet inspect --help

puppet-inspect(8) -- Send an inspection report (Deprecated)
...
```